packages/core/src/extension/extension.ts

```diff
@@ -5,7 +5,6 @@ export interface Extension {
     url: string;
 }
 
-export enum ExtensionRegion {
-    ShippingShippingAddressFormBefore = 'shipping.shippingAddressForm.before',
-    ShippingShippingAddressFormAfter = 'shipping.shippingAddressForm.after',
-}
+export type ExtensionRegion =
```

Comment:
Is there a particular reason why it's changed from enum to union type?

Reply:
We are experiencing an issue that checkout-js does not recognize SDK's enum value (enum value undefined) during compile time. Converting it to union types is a workaround for now.
cc @animesh1987

Reply:
Actually I was trying to find a solution for it, can you try using 
```
export const enum ExtensionRegion {
    ShippingShippingAddressFormBefore = 'shipping.shippingAddressForm.before',
    ShippingShippingAddressFormAfter = 'shipping.shippingAddressForm.after',
}
```

I believe this should be able to resolve the issue.

Reply:
Hrm interesting, perhaps it's exported as a type rather than a value. Perhaps we can try exporting it explicitly in bundle index file rather than relying on API extractor?

Reply:
@davidchin yes that's correct its an issue with Typescript as far as I know. Hence I suggested to try and add a `const` keyword above and that might solve the issue.

Reply:
`export const enum ExtensionRegion` is working. I'll apply it all enums.